Library/Homebrew/utils/gems.rb

```diff
@@ -14,7 +14,7 @@ module Homebrew
 
   # Bump this whenever a committed vendored gem is later added to gitignore.
   # This will trigger it to reinstall properly if `brew install-bundler-gems` needs it.
-  VENDOR_VERSION = 4
+  VENDOR_VERSION = 5
```

Comment:
@Bo98 Does this also need to be bumped each time a line is removed from `.gitignore` like we're doing? If not: let's not. If so: let's update the comment.

Reply:
With ActiveSupport we gitignore the whole gem and have exceptions for individual files.

So removing an exception from gitignore = adding to git's ignore list if that makes sense

Reply:
Gotcha.